scripts/build/cc/100-gcc.sh

```diff
@@ -403,10 +403,12 @@ do_gcc_core_backend() {
         extra_config+=("--with-system-zlib")
     fi
 
-    # Some versions of gcc have a deffective --enable-multilib.
+    # Some versions of gcc have a defective --enable-multilib.
```

Comment:
I think *deficient* would be even more accurate.
But I'm not the native english speaker... ;-)